src/driver/amdxdna/amdxdna_ctx.c

```diff
@@ -477,6 +477,13 @@ static int amdxdna_cmds_submit(struct amdxdna_client *client,
 unlock_srcu:
 	srcu_read_unlock(&client->hwctx_srcu, idx);
 	amdxdna_arg_bos_put(job);
+put_cmd_bo:
+	for (i = 0; i < args->cmd_bo_count; i++) {
+		if (!cmd_bo[i])
+			continue;
+		drm_gem_object_put(to_gobj(cmd_bo[i]));
```

Comment:
amdxdna_gem_put_obj(), please, since it is created from amdxdna_gem_get_obj().

Reply:
fixed.